recipes/keyring/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "9.0" %}
+
+package:
+    name: keyring
+    version: {{ version }}
+
+source:
+    fn: keyring-{{ version }}.tar.gz
+    url: https://files.pythonhosted.org/packages/source/k/keyring/keyring-{{ version }}.tar.gz
+    sha256: 1c1222298da2100128f821c57096c69cb6cec0d22ba3b66c2859ae95ae473799
+    patches:
+        - setup.py.patch
```

Comment:
Could we get an explanation of the patch? Looks sensible to me, but a comment here would be nice. Also, maybe open an issue upstream about adding these options.